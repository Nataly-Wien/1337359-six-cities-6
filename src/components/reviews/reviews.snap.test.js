import React from 'react';
import {render} from '@testing-library/react';
import Reviews from './reviews';

it(`Should reviews render correctly`, () => {
  const reviews = [
    {
      id: 1,
      user: {
        id: 6,
        isPro: true,
        name: `Carlo`,
        avatarUrl: `img/avatar1.jpg`
      },
      rating: 5,
      comment: `Cannot fault the hotel for cleanliness and friendliness of staff, who attended to anything we asked. The hotel facilities were great and the breakfast was really good value and lots of choice.`,
      date: `2020-11-27T09:09:35.123Z`
    },
    {
      id: 2,
      user: {
        id: 11,
        isPro: true,
        name: `Liz`,
        avatarUrl: `img/avatar2.jpg`
      },
      rating: 3,
      comment: `Excellent location, just across the river from the Louvre and couple of minutes away from dozens of cafes and restaurants.`,
      date: `2021-02-23T11:17:25.810Z`
    },
  ];

  const {container} = render(<Reviews reviews={reviews} />);

  expect(container).toMatchSnapshot();
});
