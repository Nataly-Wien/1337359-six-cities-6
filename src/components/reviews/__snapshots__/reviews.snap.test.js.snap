// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should reviews render correctly 1`] = `
<div>
  <h2
    class="reviews__title"
  >
    Reviews Â· 
    <span
      class="reviews__amount"
    >
      2
    </span>
  </h2>
  <ul
    class="reviews__list"
  >
    <li
      class="reviews__item"
    >
      <div
        class="reviews__user user"
      >
        <div
          class="reviews__avatar-wrapper user__avatar-wrapper"
        >
          <img
            alt="Reviews avatar"
            class="reviews__avatar user__avatar"
            height="54"
            src="img/avatar2.jpg"
            width="54"
          />
        </div>
        <span
          class="reviews__user-name"
        >
          Liz
        </span>
      </div>
      <div
        class="reviews__info"
      >
        <div
          class="reviews__rating rating"
        >
          <div
            class="reviews__stars rating__stars"
          >
            <span
              style="width: 60%;"
            />
            <span
              class="visually-hidden"
            >
              Rating
            </span>
          </div>
        </div>
        <p
          class="reviews__text"
        >
          Excellent location, just across the river from the Louvre and couple of minutes away from dozens of cafes and restaurants.
        </p>
        <time
          class="reviews__time"
          datetime="2021-02-23T11:17:25.810Z"
        >
          February 2021
        </time>
      </div>
    </li>
    <li
      class="reviews__item"
    >
      <div
        class="reviews__user user"
      >
        <div
          class="reviews__avatar-wrapper user__avatar-wrapper"
        >
          <img
            alt="Reviews avatar"
            class="reviews__avatar user__avatar"
            height="54"
            src="img/avatar1.jpg"
            width="54"
          />
        </div>
        <span
          class="reviews__user-name"
        >
          Carlo
        </span>
      </div>
      <div
        class="reviews__info"
      >
        <div
          class="reviews__rating rating"
        >
          <div
            class="reviews__stars rating__stars"
          >
            <span
              style="width: 100%;"
            />
            <span
              class="visually-hidden"
            >
              Rating
            </span>
          </div>
        </div>
        <p
          class="reviews__text"
        >
          Cannot fault the hotel for cleanliness and friendliness of staff, who attended to anything we asked. The hotel facilities were great and the breakfast was really good value and lots of choice.
        </p>
        <time
          class="reviews__time"
          datetime="2020-11-27T09:09:35.123Z"
        >
          November 2020
        </time>
      </div>
    </li>
  </ul>
</div>
`;
